src/java.base/share/classes/java/lang/Object.java

```diff
@@ -246,12 +253,14 @@ public boolean equals(Object obj) {
      * the unsigned hexadecimal representation of the hash code of the
      * object. In other words, this method returns a string equal to the
      * value of:
-     * <blockquote>
-     * <pre>
+     * {@snippet lang=java :
      * getClass().getName() + '@' + Integer.toHexString(hashCode())
-     * </pre></blockquote>
-     *
-     * @return  a string representation of the object.
+     * }
+     * The {@link java.util.Objects#toIdentityString(Object)
+     * Objects.toIdentityString} method returns the string for an
```

Comment:
That last sentence doesn't feel like it needs to be part of `@implSpec`, although there's definitely a connection.

Reply:
That is, a clear connection to what's being described by that `@implSpec`.

Reply:
This method has an existing apiNote and implSpec. I suspect Joe meant to add this sentence to the apiNote, not the implSpec.

Reply:
I did mean to add the cross-reference to Objects.toIdentityString to the implSpec section since earlier in the implSpec section the expression that is the basis for Objects.toIdentityString is listed.

Reply:
okay